src/phoenix/trace/semantic_conventions.py

```diff
@@ -61,6 +61,19 @@ def _missing_(cls, v: Any) -> Optional["MimeType"]:
         return None if v else cls.TEXT
 
 
+EMBEDDINGS_VECTOR = "embedding.vector"
+"""
+The embedding vector.
+"""
+EMBEDDINGS_TEXT = "embedding.text"
+"""
+In the case of textual data, the text represented by the embedding vector.
+"""
+EMBEDDINGS_MODEL_NAME = "embedding.model_name"
```

Comment:
are these plural or singular?
```suggestion
EMBEDDING_VECTOR = "embedding.vector"
"""
The embedding vector.
"""
EMBEDDING_TEXT = "embedding.text"
"""
In the case of textual data, the text represented by the embedding vector.
"""
EMBEDDING_MODEL_NAME = "embedding.model_name"
```

Reply:
addressed in [2fe71e5](https://github.com/Arize-ai/phoenix/pull/1169/commits/2fe71e55efc00378f6c7b82fa6e08905b0348bfb)